src/Microsoft.SqlTools.ServiceLayer/TableDesigner/TableDesignerService.cs

```diff
@@ -57,103 +57,89 @@ public void InitializeService(ServiceHost serviceHost)
             this.ServiceHost.SetRequestHandler(SaveTableChangesRequest.Type, HandleSaveTableChangesRequest);
             this.ServiceHost.SetRequestHandler(DisposeTableDesignerRequest.Type, HandleDisposeTableDesignerRequest);
         }
+        private Task HandleRequest<T>(RequestContext<T> requestContext, Func<Task> action)
```

Comment:
This seems like something that could be put in a more general location so others can use it too...